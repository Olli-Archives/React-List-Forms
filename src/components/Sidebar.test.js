import React from 'react';
import { shallow } from 'enzyme';
import SideBar from './Sidebar';

describe('SideBar component', ()=>{
  it('renders side bar and its children', ()=>{
    const wrapper = shallow(<SideBar><h2>im a child!</h2></SideBar>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
