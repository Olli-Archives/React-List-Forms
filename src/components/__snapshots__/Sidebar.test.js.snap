// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar component renders side bar and its children 1`] = `
<nav>
  <h2>
    im a child!
  </h2>
</nav>
`;
